script/install.sh

```diff
@@ -124,17 +124,19 @@ check_tools() {
 
 # ------------------------------------------------------------------------------
 
-mkdir -p "$ROOT"
-TMP_ROOT=$(mktemp -d "$ROOT/.installer_tmp_XXXXXX")
+init() {
```

Comment:
This needs to be done _after_ the configuration and CLI arguments are read.
Note that at the moment even if `--path` was set, the temporary directory was `~/dlang/.installer_tmp`.